binary_transparency/firmware/cmd/ft_personality/impl/ft_personality.go

```diff
@@ -32,8 +32,8 @@ import (
 	"github.com/google/trillian-examples/binary_transparency/firmware/cmd/ft_personality/internal/trillian"
 	"github.com/gorilla/mux"
 
-	_ "github.com/google/trillian/merkle/rfc6962" // Load hashers
-	_ "github.com/mattn/go-sqlite3"               // Load drivers for sqlite3
+	_ "github.com/google/trillian/merkle/rfc6962/hasher" // Load hashers
```

Comment:
This package doesn't have the side-effect registration of the hasher any more, so I think it's safe to remove this line (but we can do this in a follow-up).